app/src/views/Scenes.vue

```diff
@@ -758,6 +758,7 @@ export default class SceneList extends mixins(DrawerMixin) {
           numPages: result.numPages,
         });
         this.scenes = result.items;
+        history.pushState({}, "", "#/scenes/" + page.toString());
```

Comment:
should stick to using vue-router
```suggestion
        this.$router.push(`/scenes/${page.toString()}`);
```

Reply:
Yes and no. `$router.push` is different than `history.pushState`

https://router.vuejs.org/guide/essentials/navigation.html#router-push-location-oncomplete-onabort

`$router.push` actually navigates to the new url, but all I want is to update the url in the url bar. I couldn't find similar method in `$router`.

Reply:
But we do want to navigate, that way you can press back in the browser, it will change the pagination.  
I realize that it wasn't included in the original description but it seems logical to me. It also requires watching the route, to react to update the search according to the route change. I guess it should be another issue ?

But even then, $router.push and history.pushState should be the same, since vue-router is in hash mode. What difference are you thinking of ?

Reply:
history.pushState just pushes new state to history, so you can press back in the browser. $router.push pushes new state AND triggers router change in $router.

Reply:
But what's bad about that?

Reply:
It creates an infinite loop, where it tries to load the new page infinitely.

Reply:
I guess $router is the better solution, but using history may be a intermediate solution for now

Reply:
After taking a look at the code again, you are right @leadwolf, it's better to use the router for actual navigation. I'll need to rework this functionality.